scripts/generate_mkdocs.py

```diff
@@ -22,13 +22,14 @@ class Section(NamedTuple):
 SECTIONS: list[Section] = [
     Section("Overview", "index.md", generated=True),
     Section("Tutorial", "tutorial.md", generated=False),
-    Section("Installation", "installation.md", generated=False),
-    Section("Usage", "usage.md", generated=False),
-    Section("Configuration", "configuration.md", generated=False),
+    Section("Installing Ruff", "installation.md", generated=False),
+    Section("The Ruff Linter", "linter.md", generated=False),
+    Section("The Ruff Formatter", "formatter.md", generated=False),
+    Section("Configuring Ruff", "configuration.md", generated=False),
     Section("Preview", "preview.md", generated=False),
     Section("Rules", "rules.md", generated=True),
     Section("Settings", "settings.md", generated=True),
```

Comment:
Do you think it's useful to have a dedicated "Reference" section which would contain the "Rules" and "Settings" sub-sections?